.gitmodules

```diff
@@ -37,3 +37,6 @@
 [submodule "3rdparty/freetype"]
 	path = 3rdparty/freetype
 	url = https://github.com/freetype/freetype
+[submodule "3rdparty/JoltPhysics"]
+	path = 3rdparty/JoltPhysics
+	url = git@github.com:jrouwe/JoltPhysics.git
```

Comment:
If you didn't already do this, please make sure you track an "official" release by git-tag, if that exists for Jolt.

For all the other submodules, except the Magnum graphics related ones, I Just track whatever the latest official release tag is.

For magnum, they just kind of stopped doing releases, so i just periodically bump to the latest commit.

Reply:
It should be tracking v5.0.0 ? hopefully

Reply:
The Jolt submodule is set to use a git+ssh protocol, and this only works if your client is setup with a valid GitHub SSH credential (e.g. it cannot be cloned anonymously this way)

Like the other dependencies I'm assuming this should be using the HTTPS URI of `https://github.com/jrouwe/JoltPhysics.git` so that it can be cloned anonymously